### PR TITLE
Remove extra help command

### PR DESCRIPTION
**Pull request by ag-hcoder**
Since codemon now uses --help flag to show help, in my opinion, it's of no use to have the repetitive  "shows help". @ankingcodes  Please validate if you find this useful

### CODE REVIEW

File: codemon/CodemonHelp.py [hunk 1]
```diff
@@ -5,7 +5,6 @@ def showHelp():
   print(colored.green("           ---CODEMON---          "))
   print("A CLI tool to ace competitive programming contests")
   print(colored.cyan("\nCOMMANDS: \n"))
-  print("codemon - - - - - - - - - - - - - - -  shows help")
```
Comment (ankushbhardwxj) [inline codemon/CodemonHelp.py:8]: Instead of removing it, let's keep this line as is.

File: codemon/CodemonHelp.py [hunk 2]
```diff
@@ -5,7 +5,6 @@ def showHelp():
   print(colored.green("           ---CODEMON---          "))
   print("A CLI tool to ace competitive programming contests")
   print(colored.cyan("\nCOMMANDS: \n"))
-  print("codemon - - - - - - - - - - - - - - -  shows help")
   print("codemon --help - - - - - - - - - - - - shows help")
```
Comment (ankushbhardwxj) [inline codemon/CodemonHelp.py:9]: Change "show help" to "displays this message"

**Review by ankushbhardwxj — CHANGES_REQUESTED** [review]

File: codemon/CodemonHelp.py (hunk 2 above)
Comment (ag-hcoder) [reply codemon/CodemonHelp.py:9]: Sure

Comment (ag-hcoder) [reply codemon/CodemonHelp.py:9]: Done @ankingcodes

**Comment by ankushbhardwxj** [comment]
Thanks ! @ag-hcoder

**Pull request merged**